test/phase_info_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import openhtf
+
+
+def PlainFunc():
+  """Plain Docstring"""
+  pass
+
+
+def ExtraArgFunc(input=None):
+  return input
+
+
+class TestPhaseInfo(unittest.TestCase):
+
+  def testBasics(self):
+      phase = openhtf.PhaseInfo.WrapOrCopy(PlainFunc)
+      assert phase.func is PlainFunc
```

Comment:
Hm, that's neat to know, but pytest is really a layer up from here.  It's not likely anyone will run these tests outside of `setup.py test`, but there's no particular reason to not use base `unittest` features, I think, and it would be better to be consistent with all the other tests for now.